addons/website_event_track/tests/test_event_menus.py

```diff
@@ -11,9 +11,9 @@
 class TestEventWebsiteTrack(OnlineEventCase):
 
     def _get_menus(self):
-        return super(TestEventWebsiteTrack, self)._get_menus() | set(['Talks', 'Agenda', 'Talk Proposals'])
+        return super()._get_menus() | {'Talks'}
 
-    @users('user_eventmanager')
+    @users('user_event_web_manager')
```

Comment:
why :thinking:

Reply:
`user_eventmanager` does not have website menu access [here](https://github.com/odoo/odoo/pull/171696/files#diff-56454fcffb7f140ad5bfc994b5a57d4abe7dbcf9faaf9b3dea8409d99dc226dfR392)
Alternative would be to use `sudo` there